### DISCUSSION TITLE
[Solved] Do custom icons not support notification dots?

### DISCUSSION THREAD

**Discussion by D3m3rion** [elrumo/macOS_Big_Sur_icons_replacements#1174] [General]
[Edit] Turned out I accidentally disabled notification dots in the System Preferences. Thank you @rodgerwerner for the tip.

Hey there!

I changed the icon of Outlook via the Finder -> Get Info, but now the notification dots are not shown anymore.
I tried to google the issue and used the search function but didn't find any explanation if this is normal/expected or a bug on my end.

If you already answered this, feel free to just link the response.

Cheers!

**Comment by rodgerwerner** [comment]
If you are talking about the standard red dots in top right (Badges) ... changing the icon should have no affect on the badge at all. I have changed several icons that use badges...like "Mail" and still get badges with counts as I receive emails.

Did you check that "Badge app icon" is in fact selected in System preferences/Notifications/Outlook ? Perhaps it somehow became de-selected?

**Reply by D3m3rion** [reply]
Thank you, it actually turned off for some reason, I feel stupid now xD
it probably was turned off because I reinstalled it and accidentally hit the wrong button. :)

**Reply by cake03** [reply]
Your issue may by solved because Microsoft [Outlook](http://microsoft.com/Outlook) has a Big sur icon

**Reply by D3m3rion** [reply]
> Your issue may by solved because Microsoft [Outlook](http://microsoft.com/Outlook) has a Big sur icon

I know, that's why I wanted to change it. I don't really like it. :)
